addons-l10n/en/echo-effect.json

```diff
@@ -0,0 +1,3 @@
+{
+  "echo-effect/echo": "Echo"
```

Comment:
Did they remove the msg but not the code?

Reply:
Hmm, seems like "yes".